lib/surrealist.rb

```diff
@@ -11,6 +11,7 @@
 require 'surrealist/instance_methods'
 require 'surrealist/schema_definer'
 require 'surrealist/string_utils'
+require 'surrealist/surrealist_helper'
```

Comment:
I think this file can be named as just 'helper' as it already lies in 'surrealist' namespace.

Reply:
Will fix